owncloudApp/src/main/java/com/owncloud/android/presentation/viewmodels/settings/SettingsSecurityViewModel.kt

```diff
@@ -44,7 +45,8 @@ class SettingsSecurityViewModel(
     fun setPrefTouchesWithOtherVisibleWindows(value: Boolean) =
         preferencesProvider.putBoolean(SettingsSecurityFragment.PREFERENCE_TOUCHES_WITH_OTHER_VISIBLE_WINDOWS, value)
 
-    fun isSecurityEnforcedEnabled() = contextProvider.getBoolean(R.bool.lock_enforced)
 
     fun getBiometricsState(): Boolean = preferencesProvider.getBoolean(BiometricActivity.PREFERENCE_SET_BIOMETRIC, false)
+
+    fun isSecurityEnforcedEnabled() = contextProvider.getInt(R.integer.lock_enforced) > LockEnforcedType.DISABLED.ordinal
```

Comment:
Parse the integer to a LockEnforcedType and check if it is different from `DISABLED`. I would not rely on ordinals, it is safer to take advantage of enums. What do you think?

Reply:
Changed.